data/scripts/xml/actions.lua

```diff
@@ -66,15 +66,6 @@ local function configureActionEvent(node)
         return nil
     end
 
-    if function_ then
-        if function_ ~= "market" then
-            io.write("[Error] Invalid function attribute, check 'data/actions/actions.xml'.\n")
-            return nil
-        end
-
-        function action.onUse(player, item, fromPosition, target, toPosition, isHotkey) return player:sendEnterMarket() end
-    end
-
```

Comment:
You can also remove the `function` variable from everywhere as this was the only possible function. Or leave it as is since this is just for backwards compabitility